clients/python/tests/test_client.py

```diff
@@ -31,6 +31,27 @@ def test_register_new(mr_client: ModelRegistry):
     assert mr_api.get_model_artifact_by_params(mv.id) is not None
 
 
+def test_register_new_using_s3_uri_builder(mr_client: ModelRegistry):
+    name = "test_model"
+    version = "1.0.0"
+    uri = utils.s3_uri_from(
+        "storage/path", "my-bucket", endpoint="my-endpoint", region="my-region"
+    )
+    rm = mr_client.register_model(
+        name,
+        uri,
+        model_format_name="test_format",
+        model_format_version="test_version",
+        version=version,
+    )
```

Comment:
nit, but superflous; I believe here is missing storage_path, ie:
```suggestion
    rm = mr_client.register_model(
        name,
        uri,
        model_format_name="test_format",
        model_format_version="test_version",
        storage_path="storage/path",
        version=version,
    )
```